services/PostService.js

```diff
@@ -0,0 +1,19 @@
+const mongoose      = require('../mongoose');
+const Post          = require('../models/Post');
+
+class PostService {
+    getOnePost(post_id) {
+        return Post.find({_id: post_id});
+    };
+
+    getAllPosts() {
+        return Post.find({});
+    };
+
+    postNewPost(title, author, body) {
+        const newPost = new Post({title: title, author: author, body: body});
+        return Post(newPost).save();
```

Comment:
This is different than what I suggested

This creates 2 Posts, and saves a single one. Calling `Post(newPost)` will create another post that has the same contents as `newPost`

Note that `Post` is a class, and `save()` is an instance function on the `Post` class

Reply:
fixed